server/issue.go

```diff
@@ -539,12 +539,13 @@ func (s *Server) getPipelineCreateForDatabaseCreate(ctx context.Context, issueCr
 		SchemaVersion: schemaVersion,
 	}
 	payload.DatabaseName, payload.Statement = getDatabaseNameAndStatement(instance.Engine, c, schema)
-	bytes, err := json.Marshal(payload)
+
+	taskStatus, err := s.getPipelineApprovalPolicyForEnv(ctx, instance.EnvironmentID)
 	if err != nil {
-		return nil, fmt.Errorf("failed to create database creation task, unable to marshal payload %w", err)
+		return nil, err
 	}
 
-	taskStatus, err := s.getPipelineApprovalPolicyForEnv(ctx, instance.EnvironmentID)
+	createDatabaseTaskList, createDatabaseTaskIndexDAGList, err := createTaskListForCreateDatabase(c.InstanceID, payload, taskStatus)
```

Comment:
nit: prefer `DatabaseCreate` instead of `CreateDatabase` for consistency with the parent function name.